svg_80x80/solid.svg

```diff
@@ -0,0 +1,35 @@
+<svg width="80" height="80" viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_1_10)">
```

Comment:
Include the following code above line 2 (The same as the smaller icon)
`<rect width="100%" height="100%" fill="white"/>`